ThreeBotPackages/threefold/tfgrid_workloads/actors/workload_manager.py

```diff
@@ -192,7 +200,7 @@ def reservation_get(self, reservation_id, schema_out=None, user_session=None):
         """
         return self._reservation_get(reservation_id)
 
-    def reservations_list(self, node_id, state, epoch, schema_out):
+    def reservations_list(self, node_id, state, epoch, schema_out, user_session):
```

Comment:
I think we should have left the `reservations_list` as it was before and instead add another method to list the workload detail.
Because now we don't have any way to list the reservation at all and that was a useful method to have.